demos/common/python/openvino/model_zoo/model_api/README.md

```diff
@@ -0,0 +1,150 @@
+
+ In the example, the SSD architecture is used to predict bounding boxes on input image `"sample.png"`. The model execution is produced by `OpenvinoAdapter`, therefore we submit the path to the model's `xml` file.
+
+Once the SSD model wrapper instance is created, we get the predictions by the model in one line: `ssd_model(input_data)` - the wrapper performs the preprocess method, synchronous inference on OpenVINO™ toolkit side and postprocess method.
+
+```python
+import cv2
+# import model wrapper class
+from openvino.model_zoo.model_api.models import SSD
+# import inference adapter and helper for runtime setup
+from openvino.model_zoo.model_api.adapters import OpenvinoAdapter, create_core
+
+
+# read input image using opencv
+input_data = cv2.imread("sample.png")
+
+# define the path to mobilenet-ssd model in IR format
+model_path = "public/mobilenet-ssd/FP32/mobilenet-ssd.xml"
+
+# create adapter for OpenVINO™ runtime, pass the model path
+model_adapter = OpenvinoAdapter(create_core(), model_path, device="CPU")
+
+# create model API wrapper for SSD architecture
+# preload=True loads the model on CPU inside the adapter
+ssd_model = SSD(model_adapter, preload=True)
+
+# apply input preprocessing, sync inference, model output postprocessing
+results = ssd_model(input_data)
+```
+
+To study the complex scenarios, refer to [Open Model Zoo Python* demos](https://github.com/openvinotoolkit/open_model_zoo/tree/master/demos), where the asynchronous inference is applied.
+
+The list of Open Model Zoo demos with Model API support:
+- [BERT Named Entity Recognition Python* Demo](https://github.com/openvinotoolkit/open_model_zoo/tree/master/demos/bert_named_entity_recognition_demo/python)
+- [BERT Question Answering Python* Demo](https://github.com/openvinotoolkit/open_model_zoo/tree/master/demos/bert_question_answering_demo/python)
+- [BERT Question Answering Embedding Python* Demo](https://github.com/openvinotoolkit/open_model_zoo/tree/master/demos/bert_question_answering_embedding_demo/python)
+- [Classification Python* Demo](https://github.com/openvinotoolkit/open_model_zoo/tree/master/demos/classification_demo/python)
+- [Image Deblurring Python* Demo](https://github.com/openvinotoolkit/open_model_zoo/tree/master/demos/deblurring_demo/python)
+- [Human Pose Estimation Python* Demo](https://github.com/openvinotoolkit/open_model_zoo/tree/master/demos/human_pose_estimation_demo/python)
+- [Instance Segmentation Python* Demo](https://github.com/openvinotoolkit/open_model_zoo/tree/master/demos/instance_segmentation_demo/python)
+- [MonoDepth Python* Demo](https://github.com/openvinotoolkit/open_model_zoo/tree/master/demos/monodepth_demo/python)
+- [Object Detection Python* Demo](https://github.com/openvinotoolkit/open_model_zoo/tree/master/demos/object_detection_demo/python)
+- [Image Segmentation Python* Demo](https://github.com/openvinotoolkit/open_model_zoo/tree/master/demos/segmentation_demo/python)
```

Comment:
Why would we want to support another list of demos? We are already having trouble supporting root demos list. Don't add another

Reply:
It's suggestion from @eaidova

Reply:
I just wanted to highlight where we use model api, no need additional list if we can mark demos in common list

Reply:
I will left this list for now. The idea was to have a small description of Model API package in OMZ docs and there refer to the Model API documentation in online doc (this description might be in a different file or in demos/readme.md). So for this purpose keeping all Python demos that support Model API might be useful.

To find such demos in a huge common demos list is not convenient

Reply:
How does the list serve the purpose of having a small description of Model API. One link to an exaple demo will be enough